example/lib/main.dart

```diff
@@ -11,6 +12,39 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MainPage();
+    return MaterialApp(
+      theme: ThemeData.light(),
+      darkTheme: ThemeData.dark(),
+      home: const MyHomePage(),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({Key? key}) : super(key: key);
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  // Color _color = Colors.blue;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Padding(
+          padding: const EdgeInsets.all(50),
+          child: ColorPicker(
+            // color: _color,
+            onChanged: (Color value) {
+              // _color = value;
+              // setState(() {});
+            },
```

Comment:
Can we have a snack bar display here? I'm not sure.

Reply:
Or a vn (ValueNotifier) + vlb (ValueListenableBuilder) for `Color`. Yes, for UI changes, I personally think it's necessary.

Reply:
@AlexV525 good idea!
Like this?

```
class _MyHomePageState extends State<MyHomePage> {
  final _colorNotifier = ValueNotifier<Color>(Colors.green);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Padding(
          padding: const EdgeInsets.all(50),
          child: ValueListenableBuilder<Color>(
            valueListenable: _colorNotifier,
            builder: (_, color, __) {
              return ColorPicker(
                color: color,
                onChanged: (value) => color = value,
              );
            },
          ),
        ),
      ),
    );
  }
}
```

Reply:
Looks good.